tests/visual/scenes/graphics/bezier.scene.ts

```diff
@@ -22,21 +22,17 @@ function drawCurve(
 
     const bezierA = new Graphics();
 
-    bezierA.beginPath();
```

Comment:
Any reason for changing this test in this PR?

It doesn't seem to relate to any of the webgl 1 changes

Reply:
not really! Figured it should use the new v8 syntax whilst i was there.